meilisearch/index.py

```diff
@@ -1881,3 +1881,63 @@ def _build_url(
         if primary_key is None and csv_delimiter is None:
             return f"{self.config.paths.index}/{self.uid}/{self.config.paths.document}"
         return f"{self.config.paths.index}/{self.uid}/{self.config.paths.document}?{parse.urlencode(parameters)}"
+
+    # SEARCH CUTOFF MS SETTINGS
```

Comment:
Please move these new methods above the start of the private methods. So move `get_search_cutoff_ms`, `update_search_cutoff_ms`, and `reset_search_cutoff_ms` above the `_batch` method.

Reply:
done